src/docs/arch-hs.dox

```diff
@@ -85,7 +85,8 @@ isolation level and doesn't hold a valid snapshot.
 @section arch_hs_reconciliation History store and reconciliation
 When a dirty page is reconciled on a user file btree, the update chain is examined and the latest
 committed update is chosen as the on-disk value. All older updates are added to the history store
-table assuming they are not yet obsolete. Additionally any mixed mode timestamps will be applied.
+table assuming they are not yet obsolete. Additionally any updates without timestamps will be
+applied.
 
```

Comment:
IDK if it's auto-magic, but line wrapping looks wrong.

Reply:
Sorry, I'm missing something -- what looks wrong?

Reply:
Oh, sorry. It was my browser wrapping lines weirdly :(